composables/braintree.ts

```diff
@@ -46,62 +144,23 @@ export function useBraintree() {
   });
 
   watchEffect(async () => {
```

Comment:
Can we figure out something that solves this lint warning without suppressing it?